pkg/instance/instance.go

```diff
@@ -177,6 +179,30 @@ func (i *Instance) makeVFS() error {
 	return err
 }
 
+// AppsFS returns the hidden filesystem associated with the specified
+// application type
+func (i *Instance) AppsFS(appsType apps.AppType) afero.Fs {
+	switch appsType {
+	case apps.Webapp:
+		return i.hiddenFS(vfs.WebappsDirName)
+	case apps.Konnector:
+		return i.hiddenFS(vfs.KonnectorsDirName)
+	}
+	panic(fmt.Errorf("Unknown application type %s", string(appsType)))
+}
+
+func (i *Instance) hiddenFS(dirname string) afero.Fs {
+	fsURL := config.FsURL()
+	switch fsURL.Scheme {
+	case "file", "mem":
+		return afero.NewBasePathFs(afero.NewOsFs(),
+			path.Join(fsURL.Path, i.Domain, dirname))
+	case "swift":
+		panic("Not implemented")
```

Comment:
Do you think it will complicated to have a swift implementation for this? In particular, do you think that having a `dirname` here can make things harder?